### ISSUE TITLE
Does not recognize pluses on the next line

### ISSUE THREAD

**Issue by jcalfee** [decaffeinate/decaffeinate#114]
```
144 :         entropyStr = (new Date()).toString() + " "
145 :             + window.screen.height + " " + window.screen.width + " " 
^^^ :            ^
```

**Comment by jcalfee** [comment]
Does not recognize continuation using a plus:
```
144 :         entropyStr = (new Date()).toString() + " "+
145 :              window.screen.height + " " + window.screen.width + " " +
^^^ :             ^
```

**Comment by jcalfee** [comment]
Continuation issue here:
```
157 :         address_string = [# S L O W
158 :             Address.fromPublic(public_key, false, 0).toString(address_prefix), # btc_uncompressed
^^^ :             ^
```

**Comment by eventualbuddha** [comment]
The second one is the same as #110. The comment + array continuation is also a [CSR bug](http://michaelficarra.github.io/CoffeeScriptRedux/#try:b%20%3D%20%5B%23%20S%20L%20O%20W%0A%20%20a%0A%5D). This should theoretically be fixed when moving to the official parser (#65).

**Comment by eventualbuddha** [comment]
Fixed in v2.7.4, and probably earlier ([repl](http://decaffeinate.github.io/decaffeinate/repl/#?evaluate=false&code=entropyStr%20%3D%20(new%20Date()).toString()%20%2B%20%22%20%22%2B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20window.screen.height%20%2B%20%22%20%22%20%2B%20window.screen.width%20%2B%20%22%20%22)).
